evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -1782,3 +1782,111 @@ def test_explain_predictions_url_email(df_with_url_and_email):
         .isnull()
         .any()
     )
+
+
+@pytest.mark.parametrize("pipeline_class,estimator", pipeline_test_cases)
+def test_explain_predictions_report_shows_original_value_if_possible(
+    pipeline_class, estimator, fraud_100
+):
+    X, y = fraud_100
+    X.ww.set_types({"country": "NaturalLanguage"})
+    component_graph = [
+        "Imputer",
+        "DateTime Featurization Component",
+        "Text Featurization Component",
+        "One Hot Encoder",
+        "Standard Scaler",
+        estimator,
+    ]
+    parameters = {
+        estimator: {"n_jobs": 1},
+    }
+    pipeline = pipeline_class(component_graph=component_graph, parameters=parameters)
+
+    y = transform_y_for_problem_type(pipeline.problem_type, y)
+
+    pipeline.fit(X, y)
+
+    report = explain_predictions(
+        pipeline, X, y, indices_to_explain=[0], output_format="dict", top_k_features=20
+    )
+    expected_feature_values = set(X.ww.iloc[0, :].tolist())
+    for explanation in report["explanations"][0]["explanations"]:
+        assert set(explanation["feature_names"]) == set(X.columns)
+        assert set(explanation["feature_values"]) == expected_feature_values
+
+    X_null = X.ww.copy()
+    X_null.loc[0, "lat"] = None
+    X_null.ww.init(schema=X.ww.schema)
+
+    report = explain_predictions(
+        pipeline,
+        X_null,
+        y,
+        indices_to_explain=[0],
+        output_format="dict",
+        top_k_features=20,
+    )
+    for explanation in report["explanations"][0]["explanations"]:
+        assert set(explanation["feature_names"]) == set(X.columns)
+        for feature_name, feature_value in zip(
+            explanation["feature_names"], explanation["feature_values"]
+        ):
+            if feature_name == "lat":
```

Comment:
This is just confirming that even if our original value is nan, we don't error, right?

Reply:
Right, it'll be displayed in the report like any other feature value!